addons/viewport/src/shared/index.js

```diff
@@ -15,55 +15,63 @@ export const INITIAL_VIEWPORTS = {
       margin: '0',
       boxShadow: 'none',
     },
+    type: 'desktop',
```

Comment:
Is this property documented anywhere?

Reply:
AFAIK `Viewport` definition is not documented, so it's a good idea to start doing so

Reply:
Done